### ISSUE TITLE
SDKs should have an OSS license

### ISSUE THREAD

**Issue by victorcolombo** [kurtosis-tech/kurtosis#1549]
### Background & motivation

Right now all of Kurtosis monorepo is under BSL license, that is not an OSS license.

This is fine for Kurtosis Engine and APIC, since they are external dependencies for most systems. But since the SDK is actually incorporated in codebases, ideally it would be OSS compatible to be friendly to other OSS codebases.

This also unblocks a couple of features, like better indexing and documentation features on `go.dev`, since now they won't index much based on the license we use:

<img width="1303" alt="Screenshot 2023-10-13 at 12 15 58" src="https://github.com/kurtosis-tech/kurtosis/assets/1737857/51750a9f-4583-46df-a7e8-0ad453c8f6df">

https://pkg.go.dev/github.com/kurtosis-tech/kurtosis/api/golang@v0.84.5/engine/kurtosis_engine_rpc_api_bindings

### Desired behaviour

Leave all core Kurtosis logic under BSL, but API/SDK under an OSS license like MIT

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

### What area of the product does this pertain to?

SDK: the Software Development Kit libraries - Typescript, Go, etc.

**Comment by leeederek** [comment]
Thanks @victorcolombo - very helpful. Let me circle back internally to see if we can do a quick fix here.

**Comment by galenmarchetti** [comment]
@leeederek  this makes a ton of sense to me, I think we should do this. SDKs should be available under standard OSS

**Comment by MegaRedHand** [comment]
I think this can be closed since #2481 changed to an Apache license. Although, the version-tagged releases still seem to use BSL 🤔
https://github.com/kurtosis-tech/kurtosis/blob/28582cce6b41903b1d683126ed0f75c6503d45ff/cli/cli/.goreleaser.yml#L130
